boxes/generators/trafficlight.py

```diff
@@ -126,44 +165,46 @@ def render(self):
         self.addPart(ShadyEdge(self, None))
 
         # back
-        self.rectangularWall(th, h, "FFFF", callback=[self.backCB], move="up")
+        if self.upright:
+            self.rectangularWall(th, h, "FFFF", callback=[self.backCB, self.addMountV(th,h)], move="up", label = "back")
+        else:
+            self.rectangularWall(th, h, "FFFF", callback=[self.backCB, self.addMountH(th,h)], move="up", label = "back")
 
         if self.upright:
             # sides
-            self.rectangularWall(th, d, "fFsF", callback=[self.sideCB], move="up")
-            self.rectangularWall(th, d, "fFsF", callback=[self.sideCB], move="up")
```

Comment:
Generally these new labels that are unrelated to the new mounting holes should be in a separate commit (and may be even pull request). MAy be I do not quite care enough to split them out, though.

Reply:
Oh, I did not quite realize that this is a re-work of your first PR. Didn't intend to continue whiling about this. Sorry, no second coffee yet.